internal/db/schema/migrations/oss/postgres/91/05_deletion_tables_view.up.sql

```diff
@@ -0,0 +1,15 @@
+-- Copyright (c) HashiCorp, Inc.
+-- SPDX-License-Identifier: BUSL-1.1
+
+begin;
+  -- Originially added in 81/01_deleted_tables_and_triggers.up.sql
+  -- This is being replaced with a view.
+  drop function get_deletion_tables;
+
+  create view deletion_table as
```

Comment:
For those of us that don't know how this works, would you mind adding some comments to explain it?

Reply:
I added some comments. I would also note that this is the same query that was used by the function, just now it is used in a view.

Reply:
You think I understood it when it was a function 😂

Reply:
Fair point. Do the comments help clarify things?